src/streaming.ts

```diff
@@ -18,6 +18,8 @@ type FetchFunction = (
   identifier: string,
   environment: string,
   cluster: string,
+  // Only target-segment requests require this, and it will be safely ignored for feature calls.
+  targetSegmentRulesQueryParameter: string,
```

Comment:
It's not perfect, having this argument "ignored" by featureByIdentifier requests, but with the way this class is setup with a generic style message processor, it was the only way without a re-write.  I've confirmed using the debugger that this won't get passed in the eventual request to featureByIdentifier.